errorprone-checks/src/main/java/org/hyperledger/errorpronechecks/PrivateStaticFinalLoggers.java

```diff
@@ -61,7 +61,7 @@ public Description matchVariable(final VariableTree tree, final VisitorState sta
       return NO_MATCH;
     }
     return buildDescription(tree)
-        .addFix(addModifiers(tree, state, Modifier.PRIVATE, Modifier.STATIC, Modifier.FINAL))
+        .addFix(addModifiers(tree, state, Modifier.PRIVATE, Modifier.STATIC, Modifier.FINAL).get())
```

Comment:
is this a separate deprecation? What happens if the optional is empty?

Reply:
yes addFix(Optional) is deprecated. 
I will make it more explicit.